static/admin/config.yml

```diff
@@ -78,6 +78,8 @@ collections:
   ##
   - label: "Warrior guide"
     name: "war-guide"
+    media_folder: '/static/img/jobs/war'
```

Comment:
Is it confirmed that we are using abbreviated job names for these media folders?

Reply:
Got an answer from Nono.